src/components/charts/Periods.js

```diff
@@ -0,0 +1,138 @@
+import lodash from 'lodash';
+import { useState } from 'react';
+import styled from 'styled-components';
+import moment from 'moment';
+import WaterStats from './WaterStats';
+import useLocalStorage from '../../hooks/useLocalStorage';
+
+export default function Periods({ filteredCards }) {
+  const [timeSpan, setTimeSpan] = useLocalStorage('timeSpan');
+  const [period, setPeriod] = useState(1);
+
+  const filteredCardsByTime = filteredCards.filter(card =>
+    card.date.includes(timeSpan)
+  );
+
+  const filteredCatches = filteredCardsByTime.map(data => data.catches);
+
+  const filterNumberCatches = filteredCatches.map(number => number.length);
+
+  var numberCatches = lodash.sum(filterNumberCatches);
+
+  const m = moment();
+  const currentYear = m.format('YYYY');
+  const currentMonth = m.format('YYYY[-]MM');
+  const today = m.format('YYYY[-]MM[-]D');
+
+  return (
+    <>
+      <Header>More Stats:</Header>
```

Comment:
Looks good to me, but I'm not that into statistics, maybe you want somebody else 
to have  have a second look, too 😉